### PR TITLE
list of my projects

### PR DESCRIPTION
**Pull request by gowthamrodda**

### CODE REVIEW

File: src/components/project/project.service.ts [hunk 1]
```diff
@@ -377,6 +378,37 @@ export class ProjectService {
       },
     });
 
+    if (filter.mine) {
```
Comment (CarsonF) [inline src/components/project/project.service.ts:381]: Filters need to be applied at the database level so pagination is accurate.

File: test/utility/create-file.ts [hunk 2]
```diff
@@ -63,7 +63,6 @@ export async function createFileVersion(
   const file: CreateFileVersionInput = {
     uploadId: input.uploadId,
     parentId: input.parentId,
-    ...input,
```
Comment (CarsonF) [inline test/utility/create-file.ts:66]: Why?

File: src/components/project/project.service.ts (hunk 1 above)
Comment (gowthamrodda) [reply src/components/project/project.service.ts]: yes done

File: test/utility/create-file.ts (hunk 2 above)
Comment (gowthamrodda) [reply test/utility/create-file.ts:72]: reverted :)

**Pull request merged**